references/tags.tsv

```diff
@@ -141,20 +161,26 @@ Scotti2016_missplicing	doi:10.1038/nrg.2015.3
 Segata	doi:10.1371/journal.pcbi.1004977
 Seide2014_parallel	doi:10.1109/ICASSP.2014.6853593
 Setty2015_seqgl	doi:10.1371/journal.pcbi.1004271
+Selvaraju2016_grad  arxiv:1610.02391
 Shaham2016_batch_effects	arxiv:1610.04181
-Shrikumar2016_blackbox	arxiv:1605.01713
+Shrikumar2017_learning	arxiv:1704.02685
+Shrikumar2016_not   arxiv:1605.01713
```

Comment:
The tag `Shrikumar2016_blackbox` for `arxiv:1605.01713` is also being used in the file `04_study.md`, and maybe elsewhere.  We can either change the tag to `Shrikumar2016_not` in all files or revert to `Shrikumar2016_blackbox` here.